src/containers/ListDetails/component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import {
+  Row, Col, Typography, Spin, Empty
+} from 'antd'
+import PropsTypes from 'prop-types'
+import { MinusCircleOutlined } from '@ant-design/icons'
+
+import MoviesList from 'Components/MoviesList'
+import DeleteMovie from './DeleteMovie'
+
+const ListDetails = ({
+  onShowDeleteListModal, movies, listName, loading, isBlank
```

Comment:
use single pattern for all boolean variables: `isBlank`, `isLoading` or `blank`, `loading'